control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -456,9 +457,9 @@ func ReconcileServiceMonitor(sm *prometheusoperatorv1.ServiceMonitor, ownerRef c
 					CA: prometheusoperatorv1.SecretOrConfigMap{
 						ConfigMap: &corev1.ConfigMapKeySelector{
 							LocalObjectReference: corev1.LocalObjectReference{
-								Name: manifests.TotalClientCABundle(sm.Namespace).Name,
+								Name: manifests.RootCAConfigMap(sm.Namespace).Name,
```

Comment:
If this is an authenticated request, you'll have to specify the cert/secret which were removed in https://github.com/openshift/hypershift/pull/1896/files

Reply:
It's not an authenticated request. That's why I removed the cert/secret